examples/text-generation/utils.py

```diff
@@ -278,8 +277,9 @@ def setup_model(args, model_dtype, model_kwargs, logger):
             original_model=org_model,
             **model_kwargs,
         )
-        if not check_neural_compressor_min_version("3.2"):
-            model = model.to(model_kwargs["torch_dtype"])
+        # TODO: This will be removed in v1.19 Synapse release
+        # the loaded model should have the same dtype as original_model
+        model = model.to(model_kwargs["torch_dtype"])
```

Comment:
@libinta  we don't need this code at all for 1.19 docker, but need it for 1.18. Can we remove completely for 1.15 release?  Or do we need to have it?

Reply:
@xin3he  Can you confirm running this code on 1.19 has no effect(no harm)?

Reply:
no harm, but we'd better remove it for 1.19

Reply:
We can merge this PR into the release branch first. And once Synapse 1.19 is released, I'll cherry-pick it in main.

I agree with @jiminha that we should remove this block completely for 1.19.

Reply:
@regisss  let's merge this change now in case people are still using 1.18

Reply:
I don't understand, this change is for 1.18 compatibility?

Reply:
@regisss  the problem is it seems "version.Version(neural_compressor.__version__)" doesn't work in either 18 or 19 docker. So we can't run any INT4 quantization now with an error (AttributeError: module 'neural_compressor' has no attribute '__version__') with main.   

In synapse_8, Xin is saying we need this code "model.to(model_kwargs["torch_dtype"])" since the model is running int32  instead of bf16 without this. On 19, it seems we don't need this at all.   

Main, we still need this change since it's not working. For 19 release, we can remove this whole thing.

Reply:
Oh okay got it, so basically we should merge it into main but no need to cherry-pick this change into the release branch (since the release branch is tailored for 1.19) right?

Reply:
For the 1.19 branch, I think it's no harm to have this line, but it's better to remove it completely. I will discuss with Libin and submit patch if we want to go that way.